static/styles/6.components/nav-group.css

```diff
@@ -43,15 +43,15 @@ body.is-navigation-search .nav-group__item:not(.is-match):not(.is-match-parent)
     padding: 5px 10px;
     margin: 0 -10px;
     font-size: 16px;
-    transition: all var(--curve-fastInNormalOut) var(--duration-fast);
+    text-decoration: none;
```

Comment:
Why is text-decoration moved? In theory this changes the behavior of the link

Reply:
I like only setting animated properties for selectors like `:hover`. We aren't animating `text-decoration` so I think its clearer to set it on the "base" selector. I don't think this changes the resulting styles (at least I didn't spot a change).